### ISSUE TITLE
Feature request: Websocket support

### ISSUE THREAD

**Issue by Zetaphor** [combatwombat/netbeep#3]
First of all, thank you for making this. I've been running this full time for the past week and I'm really enjoying this augmentation to my browser experience. As someone who misses the sound of mechanical drives as a means of understanding what my computer is doing behind the GUI, this extension is greatly appreciated.

That said, it would be interesting to have an insight into Websocket connections. Would this be possible (on Firefox?). In addition to making a sound for an incoming or outgoing request, it would also be potentially interesting to have a consistent background hum to represent an open socket.

I'm interested in exploring the feasibility of this myself this weekend, but I figured I'd create an issue in case you already have some insight before I have the free time to explore this myself.

**Comment by Zetaphor** [comment]
Looking at [this extension](https://github.com/kislenko-artem/websocket_sniff), it looks like the relevant API would be `browser.runtime.onConnect`

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onConnect

**Comment by combatwombat** [comment]
Makes sense. Websockets are part of the requests, so they should make a sound. Although websocket traffic is more continous I think. Maybe a smooth, non-annoying hum for an open connection and little blips in the hum every second for data sent and received... I'll play around with that. Thanks for the feedback :)
